js/impress.js

```diff
@@ -1566,6 +1566,7 @@
             var markdownDivs = document.querySelectorAll( ".markdown" );
             for ( var idx = 0; idx < markdownDivs.length; idx++ ) {
               var element = markdownDivs[ idx ];
+              var dialect = element.dataset.extrasMarkdownDialect;
```

Comment:
Hi. You need to make this change in `src/impress.js` and then run `npm run build`.